Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Retrieve_File.yml

```diff
@@ -1,15 +1,20 @@
 contentitemexportablefields:
   contentitemfields: {}
 description: |-
-  This playbook is part of the 'Malware Investigation And Response' pack. For more information, refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
-  This playbook uses the Live Response feature to retrieve a file from an endpoint./nNote that the endpoint id will be set from the incident field "DeviceID".
+  This playbook is part of the ‘Malware Investigation And Response’ pack. For more information, refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
+  This playbook uses the Live Response feature to retrieve a file from an endpoint./nNote that the endpoint ID will be set from the incident field “DeviceID”.
```

Comment:
We need to update this PB description with your changes
Let's add that the PB now supports a supplied `machine id` as an input. Otherwise, it will take the `Device ID` incident field.
Please also mention that his PB supports only one element to be retrieved for each task (if you want more - you must use a PB loop feature).